components/authz-service/storage/v2/type.go

```diff
@@ -28,20 +28,25 @@ const (
 	Custom Type = iota
 	// ChefManaged represents a policy created by Chef Software.
 	ChefManaged
+	// System represents an underlying policy that allows Automate to function correctly.
+	// Only used here (not in API or database)
```

Comment:
Consider "System is a policy designation only used in SystemPolicies to allow OPA to function correctly."
(Mainly because I am not clear on what "only used here" means.)

Reply:
added some clarification 👍